apps/react-19-tests-v9/src/components/Dialog/Dialog.cy.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
```

Comment:
I don't think these tests are necessary, since we are reusing the tests found on the packages itself.

Reply:
it's same data set as we have for r18 v9 integration tests. want to keep consistency here although ATM we need to exclude these from main PR pipeline as their are not passing.